README.md

```diff
@@ -4,14 +4,67 @@ Library for managing [WebMock][] fixtures.
 
 [WebMock]: https://github.com/bblimke/webmock
 
-## Installation
+[Documentation](http://www.rubydoc.info/github/underdogio/webmock-fixtures).
 
+## Installation
 `gem install webmock-fixtures`
 
 or with Bundler
 
 `gem 'webmock-fixtures`
 
+## Getting Started
+WebMock fixtures provides a way to pre-define all of your fixtures ahead of time and start them easily during tests.
+
+```ruby
+require "httparty"
+require "webmock/fixtures"
+
+# Register a fixture as `:get_example`
+WebMock::Fixtures::Manager.register_fixture(
+  :get_example, :get, %r{www.example.org}, :body => "Hello World")
+
+# Create a manager with the `:get_example` fixture started
+manager = WebMock::Fixtures::Manager.run([:get_example])
+
+# Make our HTTP request
+response = HTTParty.get("http://www.example.org")
+
+# Assert we received our mock response
+if response.body != "Hello World"
+  fail("Response body \"#{response.body}\" was not what we expected")
+end
+
+# Access the created `WebMock::RequestStub` for our `:get_example` fixture
+stub = manager[:get_example]
```

Comment:
Ah, I see. Thanks for the clarification

On Wed, Nov 11, 2015 at 11:25 AM, Brett Langdon <notifications@github.com>
wrote:

> In README.md
> <https://github.com/underdogio/webmock-fixtures/pull/2#discussion_r44559437>
> :
>
> > +WebMock::Fixtures::Manager.register_fixture(
> > +  :get_example, :get, %r{www.example.org}, :body => "Hello World")
> > +
> > +# Create a manager with the `:get_example` fixture started
> > +manager = WebMock::Fixtures::Manager.run([:get_example])
> > +
> > +# Make our HTTP request
> > +response = HTTParty.get("http://www.example.org")
> > +
> > +# Assert we received our mock response
> > +if response.body != "Hello World"
> > +  fail("Response body \"#{response.body}\" was not what we expected")
> > +end
> > +
> > +# Access the created `WebMock::RequestStub` for our `:get_example` fixture
> > +stub = manager[:get_example]
>
> Yeah… so that just says that that stub has an available response to send
> someone making a request… it is not the list of responses that it has sent.
> It is just an internal list to keep track of what response should be sent.
> e.g. I can say, respond to the first two requests with “Hello World”, and
> then the third request with “Foo Bar”. On November 11, 2015 at 12:23:04 PM,
> Todd Wolfson (notifications@github.com) wrote: In README.md:
> +WebMock::Fixtures::Manager.register_fixture( + :get_example, :get, %r{
> www.example.org}, :body => "Hello World") + +# Create a manager with the
> `:get_example` fixture started +manager =
> WebMock::Fixtures::Manager.run([:get_example]) + +# Make our HTTP request
> +response = HTTParty.get("http://www.example.org") + +# Assert we
> received our mock response +if response.body != "Hello World" +
> fail("Response body \"#{response.body}\" was not what we expected") +end +
> +# Access the created `WebMock::RequestStub` for our `:get_example` fixture
> +stub = manager[:get_example]
> It totally does exist, it's known as has_responses?:
> https://github.com/bblimke/webmock/blob/v1.22.3/lib/webmock/request_stub.rb#L56-L58
> — Reply to this email directly or view it on GitHub.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/underdogio/webmock-fixtures/pull/2/files#r44559437>.
>